samples/tmo_shell/src/tmo_dfu_download.c

```diff
@@ -41,7 +41,16 @@ static char base_url_s[MAX_BASE_URL_LEN] = "https://devkit.devedge.t-mobile.com/
 
 static int iface_s = WIFI_ID; // Default iface is wifi
 
-int dfu_download(const struct dfu_file_t *dfu_file)
+char *dfu_target_str(enum dfu_tgts dfu_tgt) {
+	switch(dfu_tgt) {
+		case DFU_GECKO: return "MCU";
+		case DFU_MODEM: return "Modem";
+		default: 
```

Comment:
would prefer to use DFU_RS9116W here rather than defaulting to it. the default should return "error" or some such